pkg/core/checker.go

```diff
@@ -22,3 +22,30 @@ func (NoopChecker) Check(m Model, ops []Operation) (bool, error) {
 func (NoopChecker) Name() string {
 	return "NoopChecker"
 }
+
+type multiChecker struct {
+	checkers []Checker
+	name     string
+}
+
+func (c multiChecker) Check(m Model, ops []Operation) (valid bool, err error) {
+	for _, checker := range c.checkers {
```

Comment:
Does it's better to run concurrency?

Reply:
It's better but not necessary. That's not a RT job. :)